src/modules/IscsiLioData.rb

```diff
@@ -582,15 +582,15 @@ def GetNetConfig
     def GetIpAddr
       ip_list = GetNetConfig()
       ip_list.select! do |line|
-        line.include?("inet") && !line.include?("Scope:Link")
+        line.include?("inet") && !line.include?("deprecated") # don't show deprecated IPs
       end
 
       ip_list = ip_list.map do |ip|
         ip.lstrip!
         case ip
-        when /^inet addr: *([.\w]+)[\t ].*/
+        when /^inet *([.\w]+)\/.*/
           $1
-        when /^inet6 addr: *([:\w]+)\/.*/
+        when /^inet6 *([:\w]+)\/.*/
```

Comment:
this won't work with embedded IPv4 addresses. See: ```0:0:0:0:0:0:101.45.75.219``` (http://www.tcpipguide.com/free/t_IPv6IPv4AddressEmbedding.htm)